Applications/Spire/Include/Spire/Ui/TableModel.hpp

```diff
@@ -127,6 +149,53 @@ namespace Details {
   const T& TableModel::get(int row, int column) const {
     return std::any_cast<const T&>(at(row, column));
   }
+
+  template<typename F>
+  void TableModel::visit(const TableModel::Operation& operation, F&& f) const {
+    if(operation.type() == typeid(Transaction)) {
+      auto transaction = boost::get<Transaction>(&operation);
+      for(const auto& transaction_operation : transaction->m_operations) {
```

Comment:
Remove ```const```

Reply:
I don't understand why const should be removed. According to the express intent in the core guidelines, here const expresses the intent that the operation in the loop can't be modified.

https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#p3-express-intent

Reply:
For our codebase we don't use ```const``` other than when it's needed to satisfy an API. So for example we don't declare variables to be ```const```.